spec/fortnox/api/types/model_spec.rb

```diff
@@ -42,7 +38,7 @@
       stub_const('User', user_class)
     end
 
-    it { is_expected.not_to raise_error }
+    it { expect { User.new }.not_to raise_error }
```

Comment:
Don't love how this reads ...

Reply:
Ye, I agree. Problem is that Rubocop doesn't like when you have a proc as the `subject`. Not sure why though... I thought it was regarded as bad practice, but maybe this is something we should discuss internally.

Reply:
This is where we should use `specify` instead of `it`...